tests/pages/pages.py

```diff
@@ -1032,15 +1032,19 @@ def get_message(self, content):
 
 
 class DocumentDownloadLandingPage(BasePage):
-    continue_button = (By.CSS_SELECTOR, 'a.button')
+    continue_button = (By.CSS_SELECTOR, 'a.govuk-button')
 
     def get_service_name(self):
-        element = self.wait_for_element((By.CSS_SELECTOR, 'h1+p'))
+        element = self.wait_for_element((By.CSS_SELECTOR, 'main p:first-of-type'))
 
         return element.text.partition(' sent you ')[0]
 
     def go_to_download_page(self):
-        button = self.wait_for_element(self.continue_button)
+        try:
+            button = self.wait_for_element(self.continue_button)
+        except:
```

Comment:
Shouldn’t blanket catch all exceptions, but if it works then fine for now.
```suggestion
        except NoSuchElementException:
```